apps/editing-toolkit/editing-toolkit-plugin/premium-content/blocks/container/stripe-nudge.js

```diff
@@ -39,11 +39,11 @@ export const StripeNudge = ( { autosaveAndRedirect, stripeConnectUrl } ) => (
 			{ <Dashicon icon="star-filled" /> }
 			<span className="premium-content-block-nudge__text-container">
 				<span className="premium-content-block-nudge__title">
-					{ __( 'Connect to Stripe to use this block on your site', 'full-site-editing' ) }
+					{ __( 'Connect to Stripe to add premium content to your site.', 'full-site-editing' ) }
 				</span>
 				<span className="premium-content-block-nudge__message">
 					{ __(
-						'This block will be hidden from your visitors until you connect to Stripe.',
+						'Premium content will be hidden from your visitors until you connect to Stripe.',
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.